callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -225,5 +242,142 @@ public synchronized void sendCallStatsBridgeStatusUpdateFromQueue() {
+			final CallStatsStartConferenceListener listener) {
+		long epoch = System.currentTimeMillis() / 1000;
+		String apiTS = "" + epoch;
+		String token = getToken();
+		EventInfo event = new EventInfo(eventType.getMessageType(), "{}");
+		if (eventType == CallStatsConferenceEvents.CONFERENCE_SETUP) {
+
+			CallStatsEventMessage eventMessage = new CallStatsEventMessage(CallStatsConst.CS_VERSION, appId, CallStatsConst.END_POINT_TYPE,
+					conferenceInfo.getConfID(), apiTS, token, bridgeId, conferenceInfo.getInitiatorID(), event);
+			String requestMessageString = gson.toJson(eventMessage);
+			httpClient.sendAsyncHttpRequest(CallStatsConst.conferenceEventUrl, CallStatsConst.httpPostMethod, requestMessageString, new CallStatsHttpResponseListener() {
+				public void onResponse(HttpResponse response) {
+					int responseStatus = response.getStatusLine().getStatusCode();
+					if (responseStatus == CallStatsResponseStatus.RESPONSE_STATUS_SUCCESS) {
+						CallStatsEventResponseMessage eventResponseMessage;
+						try {
+							String responseString = EntityUtils.toString(response.getEntity());
+							eventResponseMessage = gson.fromJson(responseString, CallStatsEventResponseMessage.class);
+						} catch (ParseException e) {
+							logger.error("ParseException " + e.getMessage(), e);
+							throw new RuntimeException(e);
+						} catch (IOException e) {
+							logger.error("IO Execption " + e.getMessage(), e);
+							throw new RuntimeException(e);
+						} catch (JsonSyntaxException e) {
+							logger.error("Json Syntax Exception " + e.getMessage(), e);
+							e.printStackTrace();
+							throw new RuntimeException(e);
+						}
+						logger.debug("conference event Response status is " + eventResponseMessage.getStatus() + ":"
+								+ eventResponseMessage.getConferenceID() + ":" + eventResponseMessage.getUcID());
+						listener.onResponse(eventResponseMessage.getUcID());
+						httpClient.setDisrupted(false);
+					} else {
+						httpClient.setDisrupted(true);
+					}
+				}
+
+				public void onFailure(Exception e) {
+					logger.error("Response exception" + e.getMessage(), e);
+					httpClient.setDisrupted(true);
+				}
+
+			});
+		}
+	}
+
+	public synchronized void sendCallStatsConferenceEvents(CallStatsConferenceEvents eventType, UserInfo userInfo) {
+		long epoch = System.currentTimeMillis() / 1000;
+		String apiTS = "" + epoch;
+		String token = getToken();
+		EventInfo event = new EventInfo(eventType.getMessageType(), "{}");
+
+		CallStatsEventMessage eventMessage = new CallStatsEventMessage(CallStatsConst.CS_VERSION, appId, CallStatsConst.END_POINT_TYPE,
+				userInfo.getConfID(), apiTS, token, bridgeId, userInfo.getUserID(), userInfo.getUcID(), event);
+		String requestMessageString = gson.toJson(eventMessage);
+		httpClient.sendAsyncHttpRequest(CallStatsConst.conferenceEventUrl, CallStatsConst.httpPostMethod, requestMessageString, new CallStatsHttpResponseListener() {
```

Comment:
This code looks very similar to the one in the previous function. Can we extract common functionality into a separate function?